src/ocp_postprocess.rs

```diff
@@ -20,6 +20,7 @@ use std::{
 
 pub(crate) mod additional_trust_bundle;
 mod arguments;
+pub(crate) mod chrony_config_override;
```

Comment:
For consistency, let's remove the word override from the module's name

```suggestion
pub(crate) mod chrony_config;
```